mlflow/pytorch/__init__.py

```diff
@@ -571,3 +571,26 @@ def predict(self, data, device="cpu"):
             predicted = pd.DataFrame(preds.numpy())
             predicted.index = data.index
             return predicted
+
+
+def autolog(log_every_n_epoch=1):
```

Comment:
Can we mark this API as experimental initially to give ourselves flexibility to change it in the future based on user feedback? You should be able to do so by adding (to the top of the file):

```
from mlflow.utils.annotations import experimental
```

And then annotating this method with `@experimental`

Reply:
i have marked it as experimental.